demos/teachableMachine/index.html

```diff
@@ -86,33 +87,74 @@ <h3 class="title">LEARNING</h3>
 
   <div class="white-box output-container col-3">
     <h3 class="title">OUTPUT</h3>
-    <div class="center-container"><img id="output" src="default.png" /></div>
+    <div class="output-control">
+      <input type="radio" id="option-gif" name="output-radio" checked><label for="option-gif" class="radio-label">GIF</label>
+      <input type="radio" id="option-sound" name="output-radio"><label for="option-sound" class="radio-label">Sound</label>
+    </div>
+
+    <div id="gif-output">
+      <div class="center-container"><img id="output" src="default.png" /></div>
 
-    <div class="gif-container">
-      <div class="container" id="containerA">
-        <img class="image" id="imgA" src="output0.gif" />
-        <div class="middle" onclick="uploadGif(0)">
-          <div class="text">EDIT</div>
+      <div class="gif-container">
+        <div class="container" id="containerA">
+          <img class="image" id="imgA" src="output0.gif" />
+          <div class="middle" onclick="uploadGif(0)">
+            <div class="text">EDIT</div>
+          </div>
         </div>
-      </div>
-      
-      <div class="container" id="containerB">
-        <img class="image" id="imgB" src="output1.gif" />
-        <div class="middle" onclick="uploadGif(1)">
-          <div class="text">EDIT</div>
+        
+        <div class="container" id="containerB">
+          <img class="image" id="imgB" src="output1.gif" />
+          <div class="middle" onclick="uploadGif(1)">
+            <div class="text">EDIT</div>
+          </div>
         </div>
-      </div>
 
-      <div class="container" id="containerC">
-        <img class="image" id="imgC" src="output2.gif" />
-        <div class="middle" onclick="uploadGif(2)">
-          <div class="text">EDIT</div>
+        <div class="container" id="containerC">
+          <img class="image" id="imgC" src="output2.gif" />
+          <div class="middle" onclick="uploadGif(2)">
+            <div class="text">EDIT</div>
+          </div>
         </div>
       </div>
     </div>
+
+    <div id="sound-output" style="display:none;">
+      <div class="sound-container">
+        <div class="container" id="soundA"></div>
+        <div class="container" id="soundB"></div>
+        <div class="container" id="soundC"></div>
+      </div>
+    </div>
+
   </div>
   </div>
   <script src="sketch.js"></script>
+  <script>
+    window.onload=function() {
```

Comment:
Since we are using p5.js, maybe we could use functions from p5.dom.js.
This function can be in the `setup()` function, so we don't need window.onload. 
 Instead of`document.getElementsByTagName("label");`, we could use `selectAll('label')`. And in the for loop, we can do `labels[i].mouseClicked(toggleOutput);`